zjit/src/hir.rs

```diff
@@ -2149,6 +2161,12 @@ impl Function {
         self.infer_types();
     }
 
+    fn gen_patch_points_for_optimized_ccall(&mut self, block: BlockId, recv_class: VALUE, method_id: ID, method: *const rb_callable_method_entry_struct, state: InsnId) {
```

Comment:
Also we should skip on classes like Integer for which it's not even allowed to make singleton classes